docs/content/management-decision-logs.md

```diff
@@ -18,16 +18,35 @@ See the [Configuration Reference](../configuration) for configuration details.
 
 OPA expects the service to expose an API endpoint that will receive decision logs.
 
+#### Path Configurations
+
+There are a few ways to configure the service endpoint, either by setting `decision_logs.resource` or `decision_logs.partition_name`. See the [Configuration Reference](../configuration) for further details.
+
+##### Resource
+
 ```http
-POST /logs[/<partition_name>] HTTP/1.1
+POST /[<decision_logs.resource>] HTTP/1.1
 Content-Encoding: gzip
 Content-Type: application/json
 ```
 
-The partition name is an optional path segment that can be used to route logs
-to different backends. If the partition name is not configured on the agent,
+The resource field is an optional configuration that can be used to route logs
+to a specific endpoint in the service by defining the full path. If the resource path is not configured on the agent,
 updates will be sent to `/logs`.
 
+##### Partition Name
+
+```http
+POST /logs[/<decision_logs.partition_name>] HTTP/1.1
+Content-Encoding: gzip
+Content-Type: application/json
+```
+
+The partition name is an optional path segment that can be used to route logs
+to different backends. **Partition name will take precedence over the resource configuration option if set**. If the partition name is not configured on the agent,
+updates will be sent to the configured `decision_logs.resource` path (which defaults to `/logs`).
+
+
```

Comment:
Let's only show the "resource" path option here and omit the bit about partitions.

Reply:
got it, fixed!